pkg/apiaccess/types.go

```diff
@@ -577,8 +578,15 @@ type APIAccessUserKeyError struct {
 
 func (x *APIAccessUserKeyError) ImplementsAPIAccessKeyError() {}
 
-// EpochSeconds - The `EpochSeconds` scalar represents the number of seconds since the Unix epoch
-type EpochSeconds string
+type APIAccessKeyErrorResponse struct {
```

Comment:
This seems to have been added manually originally, needed to do the same after `tutone generate` removed